skyportal/utils/offset.py

```diff
@@ -741,13 +800,11 @@ def get_nearby_offset_stars(
             ra=source["ra"],
             dec=source["dec"],
             unit=(u.degree, u.degree),
-            pm_ra_cosdec=(
-                np.cos(source["dec"] * np.pi / 180.0) * source['pmra'] * u.mas / u.yr
```

Comment:
What's the motivation for these changes?

Reply:
This was a bug as previously written. Gaia returns the proper motion in RA in units of mas/yr, and so they have already accounted for the cosine of declination.